### ISSUE TITLE
Ambiguous member initialization: report null assignment in constructor

### ISSUE THREAD

**Issue by MazeChaZer** [kalessil/phpinspectionsea#153]
```php
<?php
class Foo
{
    private $bar;

    public function __construct()
    {
        $this->bar = null;
    }
}
```

Similar to `AmbiguousMemberInitializationInspection`.

**Comment by kalessil** [comment]
Good catch @MazeChaZer, thanks for reporting

**Comment by kalessil** [comment]
I kept thinking about the case, and come up with this case:
```php
class Clazz {
    private $property = 5;
    public function __construct() {
        $this->property = 'whatever';
    }
}
```

A new case for the immediately overridden values inspection (another one).

**Comment by funivan** [comment]
Here is another case: I think this is valid.
```php
class Clazz {
    private $property = 5;

    public function __construct() {
       if($someCondition){
         $this->property = 6;  // should be ok 
       }
    }
}
```

**Comment by kalessil** [comment]
@funivan : yep, just extended my case with following comment: `... in 1st level instructions`

**Comment by kalessil** [comment]
The `Ambiguous class property initialization` inspection has been re-branded as `Class property initialization flaws` with 2 options: `Check default values` (needs activation) and `Check constructor`.

**Comment by MazeChaZer** [comment]
Great! Thanks for the quick feedback and implementation :)
